app/grandchallenge/algorithms/models.py

```diff
@@ -419,6 +431,21 @@ def create_result(self, *, result: dict):
             )
             self.outputs.add(output_civ)
 
+    @property
+    def rendered_result_text(self):
+        interface = ComponentInterface.objects.get(slug="results-json-file")
+        try:
+            output = self.outputs.get(interface=interface)
+            if not self.algorithm_image.algorithm.result_template:
+                return output.value
+            template_output = JINJA_ENGINE.from_string(
+                self.algorithm_image.algorithm.result_template
+            ).render(dict=output.value)
```

Comment:
This creates a context environment called dict, which is a reserved built in. Could we rename it? `result` or `result_dict` maybe?